models/zone/src/index.ts

```diff
@@ -2,5 +2,5 @@ import * as ZoneDraft from './zone-draft';
 export { ZoneDraft };
 
 export { default as random } from './builder';
-export { default as presets } from './presets';
+export { default as presets } from './zone-draft/presets/sample-data-fashion';
```

Comment:
I think this file should still export the following: `export { default as presets } from './presets';` 

Perhaps the zone draft presets also need to be exported: `export { default as draftPresets } from './zone-draft/presets';`